aep/general/0158/aep.md.j2

```diff
@@ -1,100 +1,103 @@
 # Pagination
 
-APIs often need to provide collections of data, most commonly in the
-[List][aip-132] standard method. However, collections can often be arbitrarily
-sized, and also often grow over time, increasing lookup time as well as the
-size of the responses being sent over the wire. Therefore, it is important that
-collections be paginated.
+APIs often need to provide collections of data, most commonly in the [List][]
+standard method. However, collections can often be arbitrarily sized, and also
+often grow over time, increasing lookup time as well as the size of the
+responses being sent over the wire. Therefore, it is important that collections
+be paginated.
 
 ## Guidance
 
-Operations returning collections of data **must** provide pagination _at the
-outset_, as it is a [backwards-incompatible change](#backwards-compatibility)
-to add pagination to an existing method.
+RPCs returning collections of data **must** provide pagination _at the outset_,
+as it is a [backwards-incompatible change](#backwards-compatibility) to add
+pagination to an existing method.
 
-```typescript
+{% tab proto %}
+
+```proto
 // The request structure for listing books.
-interface ListBooksRequest {
+message ListBooksRequest {
   // The parent, which owns this collection of books.
   // Format: publishers/{publisher}
-  parent: string;
+  string parent = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (google.api.resource_reference) = {
+      child_type: "library.googleapis.com/Book"
+    }];
 
   // The maximum number of books to return. The service may return fewer than
   // this value.
   // If unspecified, at most 50 books will be returned.
   // The maximum value is 1000; values above 1000 will be coerced to 1000.
-  maxPageSize: bigint;
+  int32 page_size = 2;
```

Comment:
```suggestion
  int32 max_page_size = 2;
```

Also needs updating elsewhere.  `max_page_size` is a more accurate name because the server may choose to return fewer than this many results (maybe there's an overfetch + filter in memory strategy and so you often end up with 90ish results rather than the full 100). 

Switching the name of this field was actually one of the few specific things JJ (or Luke) suggested we change, because they thought the AIPs had made a mistake.